### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by Sharmaryan**
**functionality**

- add to cart 
- remove from cart
- price calculation
- quantity increment/decrement

### CODE REVIEW

**Review by preethasri — APPROVED** [review]
good to merge

File: src/context/cart-context.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      {children}
+    </CartContext.Provider>
+  );
+};
+
+const useCart = () => useContext(CartContext);
+
+export { CartProvider, useCart };
+
+// const incrementQuantity = async() => {
```
Comment (chanchal-panpaliya) [inline src/context/cart-context.js]: remove unwanted code

File: src/components/CartSummary/CartSummary.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  font-weight: light;
+}
+.price-desc {
+  display: flex;
+  justify-content: space-between;
+}
+.price-desc-desc,
+.price-desc-price {
+  display: inline;
+  margin: 5px 0;
```
Comment (chanchal-panpaliya) [inline src/components/CartSummary/CartSummary.css:29]: In your code, sometimes you use rem for margin and sometimes px. Why?

**Review by shashikantmaurya0007 — APPROVED** [review]
1.ui is good.2.code is modularized 3.good to merge

File: src/components/CartSummary/CartSummary.css (hunk 2 above)
Comment (Sharmaryan) [reply src/components/CartSummary/CartSummary.css:29]: for less than 1 rem we should use px, many reviewrs suggested me this

File: src/context/cart-context.js (hunk 1 above)
Comment (Sharmaryan) [reply src/context/cart-context.js]: sure

**Pull request merged**